aiormq/abc.py

```diff
@@ -17,29 +21,30 @@
 from yarl import URL
 
 
+connection_logger = logging.getLogger("aiormq.connection")
```

Comment:
maybe we want to use the basic one? and get child here?

https://docs.python.org/3/library/logging.html#logging.Logger.getChild

Reply:
Inplace getting logger shouldn't affect performance cause this log is really specific case